client/components/LearnerViewDirective.js

```diff
@@ -524,6 +531,9 @@ tie.directive('learnerView', [function() {
         $scope.editorContents = {
           code: ''
         };
+
+        $scope.isBrowserOnly = SERVER_URL !== null;
```

Comment:
Hmm when I seem to do that, it shows up in the non-browser version, although I'm not totally sure why. :confused: